libs/net/services/Actions/Managers/IngestActionManager`.cs

```diff
@@ -118,7 +118,7 @@ public virtual bool VerifyIngest()
     /// <returns></returns>
     public virtual bool VerifySchedule(ScheduleModel schedule)
     {
-        return VerifySchedule(GetSourceDateTime(DateTime.UtcNow), schedule);
+        return VerifySchedule(GetSourceDateTime(DateTime.Now), schedule);
```

Comment:
Using `Now` means use the system time.  In most cases this will be UTC on linux containers, but it could be something else.